data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,736 @@
+			`	After the briefing is finished, the Hai woman approaches you. "Greetings, Captain <last>. I am grateful that you will be helping us. My name is Sayari. I was the envoy to the human government a century ago." The present situation is all the evidence needed to make it clear that the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them.`
+			`	Before you can get into a conversation with Sayari, the presenter butts in. "Yes, yes, and I have a task for you to be helpful with now. Please meet me outside at the port," he says before swiftly leaving the room.`
+			`	Sayari chuckles. "So much on that human's mind. I have not seen him take a break since he has arrived. It is as if he is more stressed about this situation than any Hai."`
+				decline
+
+
+
+mission "Hai Reveal: Meet the Team: Xilin"
+	name "New Arrivals"
+	description "Travel to <stopovers> with Xilin to interview new arrivals to Hai space to determine how they arrived and what they think about the Hai. Return to <destination> when finished."
+	source "Hai-home"
+	stopover "Allhome"
+	passengers 2
+	blocked "You will need <capacity> in order to take this mission."
+	to offer
+		has "hr: meet the team"
+		has "Hai Reveal: Meet the Team: Alondo: done"
+	on offer
+		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
+		conversation
+			`Sayari follows you out to the landing pads to meet the presenter. "Captain <last>," he says while grasping your hand in a firm handshake. "Xilin Yang, Republic Deputy Minister of Foreign Affairs. I apologize for not introducing myself sooner. It's an honor to meet a hero of the Pug crisis such as yourself."`
+			choice
+				`	"It was hard-fought, but it was necessary."`
+				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
+					goto humble
+			`	"You're every inch what I expected," he assures you.`
+				goto mission
+			label humble
+			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better person than I expected to meet.`
+			label mission
+			`	"Now, about that task. I mentioned during our briefing that we need to determine how much of a threat these new arrivals might be. Not a threat in the sense that they will become violent, but in the sense that they could exacerbate the situation, such as by returning to our space and making a big stink about what they've found here. The best place to go for this would be <stopovers>. I have a sentry there keeping watch and recording new arrivals, but I'd like to meet some of them for myself."`
```

Comment:
```suggestion
			`	"Now, about that task. I mentioned during our briefing that we need to determine how much of a threat these new arrivals might be. Not a threat in the sense that they will become violent, but in the sense that they could exacerbate the situation, such as by returning to our space and making a big stink about what they've found here. The best place to go for this would be <stopovers>."`
```